### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by krate27**
Screenshot (549)
Screenshot (550)
Screenshot (551)

### CODE REVIEW

File: Components-E-Com/Home-Page/index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> Villainess-Store </title>
+    <link rel="stylesheet" href="https://villainess-cl-154.netlify.app/Components/Villainess.css"> 
+    <script src="https://kit.fontawesome.com/79dfc2e9eb.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/Components-E-Com/home.css">
+</head>
+<body>
+    <!-- Navbar  -->
```
Comment (abhij1607) [inline Components-E-Com/Home-Page/index.html:15]: you might wrap this nav in the header

File: Components-E-Com/Home-Page/index.html [hunk 2]
```diff
@@ -23,38 +25,90 @@ <h1 class="land-heading-title">
             <ul class="ul-landing-page">
                 <li class="li-landing-page"><a href="/" class="landing-page-link"> Home </a></li>
```
Comment (saadahmadhere) [inline Components-E-Com/Home-Page/index.html:26]: putting <a> in the next line would make the code more clear

File: Components-E-Com/home.css [hunk 3]
```diff
@@ -1,28 +1,95 @@
...
-    background-image: url("/wallpapersden.com_marvel-s-avengers-4k-ultra-hd_3840x2160.jpg");
+.search-box {
+    width: fit-content;
+    height: fit-content;
+    position: relative;
+    margin-right: 3rem;
+}
+
+.input-search {
+    height: 50px;
```
Comment (saadahmadhere) [inline Components-E-Com/home.css:12]: relative unit would be better

**Review by saadahmadhere — APPROVED** [review]
Suggested a few improvements. Overall code seems good.

**Pull request merged**